tests/unit/Http/SetCookieTest.php

```diff
@@ -215,15 +216,15 @@ public function testDoesNotMatchIp(): void
 
     public function testConvertsToString(): void
     {
-        $t      = 1382916008;
+        $t = 1382916008;
         $cookie = new SetCookie([
-            'Name'     => 'test',
-            'Value'    => '123',
-            'Domain'   => 'foo.com',
-            'Expires'  => $t,
-            'Path'     => '/abc',
+            'Name' => 'test',
+            'Value' => '123',
+            'Domain' => 'foo.com',
+            'Expires' => $t,
+            'Path' => '/abc',
             'HttpOnly' => true,
-            'Secure'   => true
+            'Secure' => true,
```

Comment:
What do you think about tailing comma?

Reply:
+1 for it. If you are extending such points with new items, your diff will be reduced by one line which is really useful on reading PRs.